### PR TITLE
Fix tests and disable scm environment variable injection

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: extensions/Microsoft.AspNetCore.AzureAppServices.SiteExtension/scmApplicationHost.xdt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿<?xml version="1.0"?>
+<configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
```
Comment (Tratcher) [inline extensions/Microsoft.AspNetCore.AzureAppServices.SiteExtension/scmApplicationHost.xdt]: what does this file do?

Comment (pakrym) [reply extensions/Microsoft.AspNetCore.AzureAppServices.SiteExtension/scmApplicationHost.xdt]: By being there it prevents main transformation file from applying to scm host.

**Comment by Tratcher** [comment]
Your PR descriptions are lacking of late.

File: extensions/Microsoft.AspNetCore.AzureAppServices.SiteExtension/scmApplicationHost.xdt (hunk 1 above)
Comment (Tratcher) [reply extensions/Microsoft.AspNetCore.AzureAppServices.SiteExtension/scmApplicationHost.xdt]: Add an XML comment to that effect

**Pull request merged**